proto/api.proto

```diff
@@ -269,7 +270,7 @@ message EndTransactionResponse {
   // Remaining time (ns).
   optional int64 commit_wait = 2 [(gogoproto.nullable) = false];
   // List of intents resolved by EndTransaction call.
-  repeated bytes resolved = 3 [(gogoproto.customtype) = "Key"];
+  repeated bytes resolved = 3 [(gogoproto.casttype) = "Key"];
```

Comment:
It seems to be implied by `casttype`, which is what https://github.com/gogo/protobuf/issues/64 was about. Separately, it's implied by `repeated`.